src/System.IO.IsolatedStorage/tests/project.json

```diff
@@ -32,9 +34,19 @@
     "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0040"
   },
   "frameworks": {
-    "netstandard1.7": {}
+    "netstandard1.7": {
+      "imports": [
```

Comment:
Why did you add this?  Imports should never be needed for our tests since they should be picking up the latest packages which all support netstandard.